lib/libzfs/libzfs_sendrecv.c

```diff
@@ -1007,8 +1007,17 @@ send_print_verbose(FILE *fout, const char *tosnap, const char *fromsnap,
 			(void) fprintf(fout, dgettext(TEXT_DOMAIN,
 			    "incremental\t%s\t%s"), fromsnap, tosnap);
 		} else {
+#if defined(__GNUC__) && !defined(__clang__) && \
+	defined(HAVE_FORMAT_OVERFLOW)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-overflow"
+#endif
```

Comment:
What does the compiler print when we do not do this?

I am curious since there are other fprintf() calls in the same function that do not receive this treatment.